components/PasswordDialog.qml

```diff
@@ -352,7 +353,8 @@ Item {
                     }
                 }
 
-                Keys.onReturnPressed: {
+                onAccepted: {
+                    if (!root.visible) return
```

Comment:
Why do you need this?

Reply:
Wanted to avoid something like #1512

Reply:
Would prefer not to do this

Reply:
Everywhere or just here?

Reply:
Everywhere. Let me know if this will lead to a bug, would like to reproduce it.

Reply:
Yes, it leads to a bug. You can try yourself: Open GUI, go to Transactions -> Payment proof -> Press enter.